input/redis/README.md

```diff
@@ -22,6 +22,13 @@ input:
 
     # (optional) BLPOP blocking timeout, default: "600s"
     blocking_timeout: "600s"
+    
+    # (optional) Password for DB, default: null
```

Comment:
The type of `password` is `string`, so I think the default value is an empty string `""`.